nss-day-cohort-48/giffygram-archies-bunker2-0
# Cjw create posts

#### Changes Made
1. Added file PostEntry.js  to feed directory.
1. Modified file PostEntry.js to include PostEntry function and an event listener that  sends new post in sendPost function.
​
#### Steps to Review
1. Checkout this branch locally.
    
    git fetch --all
    git checkout cjw-create-posts
    
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > code .
   > cd src
  > serve
  > check in PostEntry.js for PostEntry function
  > check in PostList.js for PostList function that renders html
  > check in GiffyGram.js for invoked PostEntry and PostList
  > check in browser for new posts

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
